src/components/Footer.js

```diff
@@ -1,11 +1,40 @@
-import React from 'react';
+//THE BELOW COMMENT IS REQUIRED TO MAKE emotion/react WORK!!!!
+//https://www.digitalocean.com/community/tutorials/react-react-emotion
+/** @jsxImportSource @emotion/react */
+import { css } from '@emotion/react';
+
+export default function Footer(props) {
+    if (Object.keys(props).length === 0) {
+        return;
+    }
+    const color = '#aeaeae';
+    const container = css`
+        align-items: center;
+        color: ${color};
+        display: flex;
+        flex-direction: column-reverse;
+        justify-content: center;
+        row-gap: 1rem;
+        @media screen and (min-width: 48rem) {
+            flex-direction: row;
+            justify-content: space-between;
+        }`
+    const nav = css`
+        column-gap: 1rem;
+        display: flex;`
+    const anchor = css`
+        color: ${color};
+        text-decoration: none;`
 
-export default function Footer() {
     return (
-        <div>
-            &copy; 0000 UIsual
-            ...
-            ...
+        <div css = {container}>
```

Comment:
```suggestion
        <div css={container}>
```

Asegurate de aplicar un formateador de code tipo prettier antes de subir codigo para alinear el formato del codebase en todos lados